XCUITests/AuthenticationTest.swift

```diff
@@ -60,6 +60,7 @@ class AuthenticationTest: BaseTestCase {
     }
     
     fileprivate func enablePasscode(_ passCode: String) {
+        navigator.goto(NewTabMenu)
```

Comment:
It's probably something to do with screengraph is always set to find the shortest path I suppose.  My guess is that the state that AuthenticationTest was in before calling enablePassword may be closer to where it went, rather than the NewTabMenu scene.